pkg/commands/create.go

```diff
@@ -18,10 +18,18 @@ type CreateArgs struct {
 	Tags       []string
 	Volumes    []string
 	Image      string
+	TmpSSHKey  bool
 }
 
 // RunCreate is the handler for 'scw create'
 func RunCreate(ctx CommandContext, args CreateArgs) error {
+	if args.TmpSSHKey == true {
```

Comment:
$> golint pkg/commands/create.go
$>

I have no error, but I can change if you want 😊